docs/docs/sample-scripts/teams/export-teams-conversations.md

```diff
@@ -0,0 +1,95 @@
+# Export conversations from Microsoft Teams Channels
+
+Author: [Joseph Velliah](https://sprider.blog/export-conversations-from-microsoft-teams)
+
+## Problem Statement
+
+We utilize Teams during incidents and create channels for each. We would like to be able to export conversation history.
+
+- Right now the only option we have is to go through Security & Compliance.
+- Teams usage is growing in every organization and it would soon become unreasonably to only have Administrators be the ones doing exports of channels for all the Teams.
+
+### Solution
+
+This script uses Microsoft 365 CLI to export the conversations from Microsoft Teams Channels.
+
+!!! notes
+
+- Commands m365 teams message list and reply list are based on an API that is currently in preview and is subject to change once the API reached general availability.
+- You can only retrieve a message from a Microsoft Teams team if you are a member of that team.
```

Comment:
This is a good statement and I'd pair it up with the previous statement about admins to make it clear that you don't need to be a tenant admin to export conversations but you still can only export conversations of Teams of which you're member.